NSS-Day-Cohort-68/bangazon-client-hedgehogs
# Client Side | Feature client like prods

Authenticated user can like and unlike products. Like products will display on the user profile, Unliked products will be removed from the user profile. 

### Changes

- Added` [isLiked, setIsLiked] `to state on `components/product/details.js`
- Added functions for `like` and `unlike` as well as `useEffects` for `likedProduct` and `getUserProfile`

### Testing

- Log into the Bangazon application as a User (ex. Steve)
- Open dev tools in the application
- Go to **Products**, then click on a product to pull up the **product details** view
- Click on the **Like Product** button
- Network tab in dev tools should reflect that you selected a product and liked it
- You should also see the like button change to 'unlike' as the next option for the product by the user
- Go to the **user profile**, using the drop-down on the right, where you should see product tile(s) for liked products
- Go back to a liked product and **unlike** it, then return to the user profile where you should no longer see that product in the liked list

### Related Issue

- Fixes Ticket # 20
